### ISSUE TITLE
`th` was passed a style object that has previously been mutated.

### ISSUE THREAD

**Issue by strivek** [AllenFang/react-bootstrap-table#155]
in  my project  td width style has a  ** NaN** value and style not show correct , console has a log
```
Warning: `th` was passed a style object that has previously been mutated. Mutating `style` is deprecated. Consider cloning it beforehand. Check the `render` of `TableBody`. Previous style: {display: {}, width: NaN, maxWidth: NaN}. Mutated style: {display: {}, width: NaN, maxWidth: NaN}.
```
how to fix width is NaN?

**Comment by AllenFang** [comment]
ok, I'll fix this warning.

**Comment by AllenFang** [comment]
Fix this issue on v1.2.14
